feed/admin.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
-from django.contrib import admin
-from .models import BlogPost 
+from .models import BlogPost
+from comment.models import Comment
```

Comment:
Can be removed. `Comment` isn't being used in this file. You can always try this too:

```bash
pip install flake8
flake8 myfile.py
```

That will tell you all the ways you can fix up your code including removing unused imports.